app/controllers/api/v1/wallet_controller.rb

```diff
@@ -25,6 +27,19 @@ def show
         end
       end
 
+      def update
+        return error_handler if params[:id].blank?
+        wallet = Wallet.find_by(id: params['id'], status: 1, deleted_at: nil)
```

Comment:
what do you think of adding a method to the model like this:
find_active_wallet_by_id
and here you call
Wallet.find_active_wallet_by_id(params['id'])